pages/linux/nixos-rebuild.md

```diff
@@ -0,0 +1,31 @@
+# nixos-rebuild
+
+> Reconfigure a NixOS machine.
```

Comment:
Perhaps we could add an additional line below this one that links to the documentation with the new homepage syntax we drafted up in #2649?

```
> Documentation: https://nixos.org/nixos/manual/#sec-changing-config
```

Reply:
Okay, I just committed the above change

Reply:
Cool! I've just realised that I got the syntax incorrect in my above comment. I've made quick commit to fix it, if you don't mind?

Reply:
No problem at all. Thanks!